published/AWS/servers_by_account_and_tags/function.js

```diff
@@ -0,0 +1,119 @@
+function(account_label, env_tag_value, ellipsis) {
+  "use strict"; 
+const Q = require('q');
+var AWS = require('aws-sdk');
+var _ = require('underscore');
+
+// -----------------------------------------------------------------------------------------------
+// INPUTS
+// -----------------------------------------------------------------------------------------------
+
+// Add here the region codes of the regions you want Ellipisis to search for Instances
```

Comment:
"Add here the" reads a bit awkward in English — I'd suggest:

"Add the region codes for the regions you want Ellipsis to search for instances."